### DISCUSSION TITLE
Unable to accept invitations from our company

### DISCUSSION THREAD

**Discussion by RafeedhaMohammad-Lotus** [community/community#82168] [New to GitHub]
Hello Team, we are unable to accept invitation, we are facing below error,

"You cannot accept this invitation because your account has been flagged. If you believe this is a mistake, please contact support."
<img width="960" alt="GIT1" src="https://github.com/community/community/assets/154888713/98dbc1c5-65d9-4e45-8e17-8c21041ef64c">

Can you please help on this? ASAP.

Thanks,
Rafeedha

**Comment by queenofcorgis** [comment]
Hi @RafeedhaMohammad-Lotus ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
